src/app.js

```diff
@@ -1,21 +1,39 @@
+/* eslint-disable no-console */
+
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const PORT = 3001;
+
+const server = http.createServer((req, resp) => {
+  resp.setHeader('Content-Type', 'application/json');
+
+  const url = new URL(req.url, `http://${req.headers.host}`);
+
+  const pathToFile = url.pathname.slice(6) || 'index.html';
+
+  if (!url.pathname.startsWith('/file') && url.pathname !== '/') {
+    resp.statusCode = 400;
+
+    resp.end('You should write "/file" in the beginning');
+
+    return;
+  }
+
+  if (!pathToFile) {
+    resp.statusCode = 404;
+    resp.end('File not found');
+  }
+
+  fs.readFile(`./public/${pathToFile}`, (error, data) => {
+    if (error) {
+      resp.statusCode = 404;
+      resp.end('File not found');
+    }
+
+    resp.end(data);
+  });
```

Comment:
This is not a requirement, just for your information.

Would be better to use streams here and pipe from `fs.createReadStream(...)` to resp (resp implements writable stream)

Reply:
Again, this is not required to do here